pallets/pool-system/src/lib.rs

```diff
@@ -165,14 +167,29 @@ pub type PoolEssenceOf<T> = PoolEssence<
 	<T as Config>::MaxTokenSymbolLength,
 >;
 
+#[derive(Encode, Decode, TypeInfo, PartialEq, Eq, MaxEncodedLen, RuntimeDebug)]
+#[repr(u32)]
+pub enum Release {
+	V0,
+	V1,
+}
+
+impl Default for Release {
+	fn default() -> Self {
+		Self::V0
+	}
+}
```

Comment:
Can we use the new versioning that is automatically supported for pallets. Cc @wischli for how this works ^^

Reply:
My understanding of that is that its based around the crate's version number. We... don't really use/maintain those right now. I agree we should move to the new versioning, though.

My thought was that we'll need to figure out some sort of versioning / release-stream system anyway once pools are on cent-chain (because we won't want new features to land simultaneously on altair & cfg). We should roll pallet versioning into that, and then we can switch to the new pallet version storage. WDYT? (again /cc @wischli )

Reply:
I think it is a basic u16 value.